pkgs/development/compilers/rust/rustc.nix

```diff
@@ -309,6 +309,9 @@ stdenv.mkDerivation (finalAttrs: {
       directory = "vendor"
       EOF
     ''
+    + ''
+      export RUSTFLAGS="$RUSTFLAGS --remap-path-prefix ''${NIX_BUILD_TOP:-/build}=/build"
```

Comment:
Should I comment why we use `$RUSTFLAGS` here, but `.cargo/config.toml` in the setup hook?

Reply:
Does the undocumented `RUSTC_DEBUGINFO_MAP` environment variable I have found do what I think it does for the rustc build?

Reply:
Indeed!

Reply:
Good catch.